crowbar_framework/config/locales/crowbar/en.yml

```diff
@@ -442,6 +442,7 @@ en:
       powercycle: 'Power Cycle'
       poweroff: 'Power Off'
       identify: 'Identify'
+      blockui_message: 'Rebooting nodes...'
```

Comment:
Is it nodes, or just one node? I believe Power action is only available from 1 node view....